### PR TITLE
refactor: cleanup unused code and add init.sql

### PR DESCRIPTION
**Pull request by hi-rustin**
The Contributions API is no longer used.

### CODE REVIEW

**Comment by hi-rustin** [comment]
/assgin @Mini256 

PTAL~

**Comment by hi-rustin** [comment]
/assign @Mini256 

PTAL~ Thanks!

File: docker-compose.yml [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
 services:
-  ti-community-bot:
-    build: ./
-    image: ti-community-bot:latest
-    ports:
-      - 3100:3100
+  db:
+    image: mysql:5.7
+    environment:
+      MYSQL_ROOT_PASSWORD: 'password'
```
[the remark this answers is not in the document]
Comment (hi-rustin) [reply docker-compose.yml:7]: > Perhaps this is more suitable as an example file and ignoring docker-compose.yml.

I think this is OK as a default password for test database, and this file is mainly used to create the database for the development environment.



> Is the ti-community-bot service missing?

I will add it in the next PR (which will complete the automation of the image build).

**Review by Mini256 — APPROVED** [review]

**Pull request merged**